### PR TITLE
Update Roslyn package versions to match Microsoft.DotNet.ProjectModel

### PR DESCRIPTION
**Pull request by prafullbhosale**
Addresses #190 
cc @JunTaoLuo

### CODE REVIEW

File: src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "Microsoft.AspNetCore.Hosting": "1.0.0-*",
     "Microsoft.AspNetCore.Server.WebListener": "0.1.0-*",
-    "Microsoft.CodeAnalysis.CSharp": "1.1.0-rc1-20151109-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.2.0-beta1-20160202-02",
```
Comment (JunTaoLuo) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: Curious, isn't `Microsoft.CodeAnalysis.CSharp` pulled in transitively from `Microsoft.Extensions.CodeGeneration.Templating`?

Comment (JunTaoLuo) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: Actually now I take a look, isn't there a lot of redundant transitive dependencies among the projects here?

Comment (prafullbhosale) [inline src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/project.json]: Yeah you are right. I'll try to get rid of the transitive dependencies.

**Comment by prafullbhosale** [comment]
The roslyn packages updates have already been checked in to dev. 
Closing the PR. 
Will migrate the other changes to another PR.

**Pull request closed without merging**